### ISSUE TITLE
A new feature 

### ISSUE THREAD

**Issue by Aashu-12** [vic1707/AutoPlay-PokeMeow-Discord#11]
This is just a kind of suggestion i wanna give 
Can u add such feature in the bot that it do premierball to full odds shiny and do mb only to mythical legendries and not cheap legenderies 
Plz add this if possible

**Comment by vic1707** [comment]
Already tried to do it but unfortunately since testing odd shiny is by definition really hard and the fact that because of them premier balls and masterballs can conflicts with each others when selecting the ball I decided to make the bot the way it is now.

As for mb on mythical legendries, because the message from pokemeow isn't different from the other kind we can't differentiate them. We could introduce a simple list (like for legendaries when fishing) but it's boring and wouldn't benefit to everyone I think.

**Comment by pokefan2** [comment]
Is it possible to make fishing up a legendary or a shiny throw Dive Balls? Since they have very high catch rate already.

**Comment by vic1707** [comment]
I'd want to do this but since I don't handle buying them it's in standby for now.

**Comment by Aashu-12** [comment]
ok 
But for mythicals the line of the embed is different so if u can do something using that or i can give u a list of mons which are mythicals and if u can add that when these mons come the bot will do mb

**Comment by vic1707** [comment]
I certainly can do something, I'll need screenshots on how the message is to identify differences with classics pokemons before anything else.

**Comment by Aashu-12** [comment]
![Screenshot (860)](https://user-images.githubusercontent.com/83766290/120893548-7c894480-c631-11eb-9642-160789db2a72.png)
like this is a mythical pokemon and its valuable so we need to mb it 
the line of the embed is red in this case 
there are other mythicals whose embed line is not red but red is there in many of them 
so this is an example 
if u can try making that if the line of the embed have this color or if the bot can detect the name of the mythical mons

**Comment by vic1707** [comment]
the only difference is the color of the embed between a mythical and a non mythical ? (didn't used the bot for a long time since my last ban).
If it is do you know if all non mythical have a certain color and all non mythical have another one (in that case it should be possible to check if the embed color is the non-mythical one and act in consequences)

**Comment by Aashu-12** [comment]
there are many mythicals 
the event mythicals are always red 
the other mythicals which can be obtained any time have different color i will give u color of all mythicals if u add the feature in it 
and all non mythicals have purple color of embed
i wanna ask that if u can make something like the user will add a list of legends which have to be mbed and rest will be ultraballs 
if this is possible it would be the best

**Comment by vic1707** [comment]
sorry I didn't noticed your answer. 

Then if all colors available are red for event mythical, purple for non mythical, and some others for classic mythical it should be possible to make some kind of filter based on the colors...

as for a whitelist like system, again it should be possible in order to save masterballs 🤔. I'm just wondering how it should be input by the user...

**Comment by Aashu-12** [comment]
yeah if u say i will post the color of all mythicals which need to be mbed 
for the thing that the user will enter names of mons i was talking about something which i saw in other selfbot not only pokemeow selfbot but some others too which is configuration file. in that we enter all the details in that config file and then the bot works accordingly so if u can do something like that 😁😁

**Comment by vic1707** [comment]
> yeah if u say i will post the color of all mythicals which need to be mbed
> for the thing that the user will enter names of mons i was talking about something which i saw in other selfbot not only pokemeow selfbot but some others too which is configuration file. in that we enter all the details in that config file and then the bot works accordingly so if u can do something like that gringrin

do you have some links to those self bots ?

I'm in holidays and wanted to fix everything and add every feature requested but currently I'm facing many many problem with the bot and discord :sweat:
